pkg/accounting/tally/config.go

```diff
@@ -26,11 +27,11 @@ func (c Config) initialize(ctx context.Context) (Tally, error) {
 	pointerdb := pointerdb.LoadFromContext(ctx)
 	overlay := overlay.LoadServerFromContext(ctx)
 	kademlia := kademlia.LoadFromContext(ctx)
-	db, err := accounting.NewDb(c.DatabaseURL)
+	db, err := accountingdb.New(c.DatabaseURL)
 	if err != nil {
 		return nil, err
 	}
-	return newTally(zap.L(), db, pointerdb, overlay, kademlia, 0, c.Interval)
+	return newTally(ctx, db, pointerdb, overlay, kademlia, 0, c.Interval, zap.L())
```

Comment:
egon previously asked me to keep the logger towards the front for whatever reason.  I guess context goes first though.